db/db_bolt.go

```diff
@@ -76,9 +76,13 @@ func (b *boltDB) Get(namespace string, key []byte) ([]byte, error) {
 		if bucket == nil {
 			return errors.Wrapf(ErrNotExist, "bucket = %s doesn't exist", namespace)
 		}
-		if value = bucket.Get(key); value == nil {
+		v := bucket.Get(key)
```

Comment:
Don't understand the difference here. What is going to be fixed?

Reply:
The mem address pointed by v is only safe within the tx lifecycle, so that I need to copy the bytes into another one that could still be safely used after the tx

Reply:
Discussed offline. There are better ways to fix but this one is good for the short term.